debian/bullseye/foreman/rules

```diff
@@ -34,7 +34,7 @@ build:
 	/bin/sed -i '2i  gem "puma-status"' bundler.d/service.rb
 	$(BUNDLE) install
 	$(BUNDLE) package --no-install
-	/usr/bin/python script/filter-package-json.py
+	/usr/bin/python3 script/filter-package-json.py
```

Comment:
It has a shebang. Should we rely on that? And perhaps long term also rewrite it to Ruby so we can drop Python as a build dependency?

Reply:
Ah, that's a fair point, I think the explicit `/usr/bin/python` was to *override* the shebang… Let's try that.

Reply:
Heh. The file is not executable. So won't work just now. I'll leave it with the explicit python3 and we'll revisit once we rewrite.